src/CONST.ts

```diff
@@ -3683,8 +3683,9 @@ const CONST = {
                     type: 'createWorkspace',
                     autoCompleted: true,
                     title: 'Create a workspace',
-                    subtitle: '<strong>Create a workspace</strong> to track expenses, scan receipts, chat, and more.',
-                    message:
+                    description:
+                        '<strong>Create a workspace</strong> to track expenses, scan receipts, chat, and more.\n' +
```

Comment:
Not sure it really matters since it works as it is and people have real reason to edit the description, but it appears that we block from using HTML tags when manually setting a task description.

<img width="392" alt="image" src="https://github.com/Expensify/App/assets/2229301/0b0e6710-0fc8-45a0-91cb-195a23297ab3">

Whereas markdown syntax is allowed and also rendered correctly
<img width="650" alt="image" src="https://github.com/Expensify/App/assets/2229301/382c5f28-765c-4c37-80d1-b5a7ee414d07">

Reply:
I think this is a good point. @rezkiy37 @rayane-djouah do you think we could look into fixing this?